build.rs

```diff
@@ -5,10 +5,17 @@
 use std::env;
 use std::process::{Command, Stdio};
 
+
+fn find_make() -> String {
+    Command::new("gmake").status()
+                         .map(|_| "gmake".to_string())
+                         .unwrap_or("make".to_string())
```

Comment:
I'll let @mbrubeck do the final review, though if it works it's probably ok.

For reference, [this](https://github.com/servo/servo/blob/master/components/style/build.rs#L26) is what it's used to find python in the style crate, though your version looks way cleaner.